contracts/tokens.js

```diff
@@ -88,6 +90,9 @@ actions.createSSC = async () => {
       await cancelBadUnstakes();
```

Comment:
Let's clean this code up now that the backfill for unstake is done while we are here

Reply:
done, removed function and unused params fields